plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -210,6 +210,16 @@ SecRule REQUEST_FILENAME "@endsWith /index.php" \
             "t:none,\
             ctl:ruleRemoveTargetById=942100;ARGS"
 
+# Cannot update page|post in WordPress due to `x-http-method-override` header
```

Comment:
IMO, this rule should go into the `*-config.conf` file, which will be loaded first.

Reply:
We can let this one pass as there are no other rules in that file relating to this.

Reply:
![image](https://github.com/coreruleset/wordpress-rule-exclusions-plugin/assets/86168235/e6250c9f-5445-418f-af80-7478e362a2db)

LOL. Merge perms on my own PR. 😂

Reply:
Fair enough.